### PR TITLE
PLG-186-Shipper Changes

### PR DESCRIPTION
**Pull request by AnandPaladin**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List
any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Chore (no code changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My commit message/PR follows the contribution guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# **Other information**:

List any documentation updates that are needed for the Wiki

### CODE REVIEW

File: jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/AzureFetchOrchestrator.java [hunk 1]
```diff
@@ -71,9 +71,9 @@ public Map<String, Object> orchestrate(){
 			fileGenerator.generateFiles(subscriptions,filePath);
 			log.info("End : FIle Generation");
 			
-			log.info("Start : Backup Current Files");
-			s3Uploader.backUpFiles(s3Bucket, s3Region, s3Data, s3Processed+ "/"+ new SimpleDateFormat("yyyyMMdd-HHmmss").format(new Date()));
-			log.info("End : Backup Current Files");
+//			log.info("Start : Backup Current Files");
```
Comment (mkinchin-paladin) [inline jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/AzureFetchOrchestrator.java:74]: Please add comment about WHY it is commented out with task name and ID

**Review by mkinchin-paladin — APPROVED** [review]
LGTM, @arunpaladin @santhosh-challa  Please folks, do review as well

**Review by arunpaladin — APPROVED** [review]

File: jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/Main.java [hunk 2]
```diff
@@ -1,78 +1,216 @@
...
+    public static void doBackUpAndCleanUpInventory(String dataSource, String srcFolder, AWSCredentialProvider awsCredentialProvider) {
+        //accessKey = System.getenv("ACCESS_KEY");
+        //secretKey = System.getenv("SECRET_KEY");
+        LOGGER.info("Inside doBackUpAndCleanUpInventory Method:{}");
+        LOGGER.debug(dataSource);
+        LOGGER.debug(srcFolder);
+
+        try {
+//            String account = "500559730414";
+//            String region  = "us-east-1";
```
Comment (santhosh-challa) [inline jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/Main.java:133]: these comments can be removed.

**Review by santhosh-challa — APPROVED** [review]
Rest LGTM

File: jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/Main.java (hunk 2 above)
Comment (AnandPaladin) [reply jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/Main.java]: removed these comments in latest push

**Review by mkinchin-paladin — COMMENTED** [review]
PR titles must be meaningful and descriptive, as well and details of PR

**Pull request merged**